wherehows-web/app/routes/application.js

```diff
@@ -6,7 +6,8 @@ const {
   Route,
   run,
   get,
-  inject: { service }
+  inject: { service },
+  testing,
```

Comment:
I see. Using `testing` seems okay but please use `transitionTo` instead of `location.replace`.
In SPA, replace the url requires extra cost to reload the all resources.